foreign-key.md

```diff
@@ -318,3 +318,26 @@ Create Table | CREATE TABLE `child` (
 ### Compatibility with MySQL
 
 When you create a foreign key without specifying a name, the name generated by TiDB is different from that generated by MySQL. For example, the foreign key name generated by TiDB is `fk_1`, `fk_2`, and `fk_3`, while the foreign key name generated by MySQL is `table_name_ibfk_1`, `table_name_ibfk_2`, and `table_name_ibfk_3`.
+
+Both MySQL and TiDB parse but ignore "inline `REFERENCES` specifications". Only `REFERENCES` specifications that are part of `FOREIGN KEY` definitions are checked and enforced. For example:
+
+```sql
+CREATE TABLE parent (
+    id INT KEY
+);
+
+CREATE TABLE child (
+    id INT,
+    pid INT REFERENCES parent(id)
+);
+
+SHOW CREATE TABLE child;
++-------+-------------------------------------------------------------+
+| Table | Create Table                                                |
++-------+-------------------------------------------------------------+
+| child | CREATE TABLE `child` (                                      |
+|       |   `id` int(11) DEFAULT NULL,                                |
+|       |   `pid` int(11) DEFAULT NULL                                |
+|       | ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
++-------+-------------------------------------------------------------+
+```
```

Comment:
In in uppercase, `SHOW CREATE TABLE` statement output indicates that no foreign keys exist in table child.

Please add this into doc, I miss it before, sorry...

Reply:
The last three line should be this:

<img width="792" alt="image" src="https://user-images.githubusercontent.com/26020263/228557540-b97772ea-4590-4ac8-acac-723946e1a054.png">

Reply:
you just delete the ``` which we should keep it.